### PR TITLE
Correcting syntax

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

File: mountainduck/issues/index.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ It may be that Finder closes the connection because Mountain Duck hasn't answere
 
 This is an issue within the operating system that can occur to any network drive.
 
-<del> **Additional `._*` Files Saved on Remote Volumes** </del> <br/>
```
Comment (dkocher) [inline mountainduck/issues/index.md:256]: Wasn't this formatted with strikethrough because no longer applicable?

Comment (Cassie205) [reply mountainduck/issues/index.md:256]: Strikethrough the header is not very user friendly as it might be difficult to find

Comment (dkocher) [reply mountainduck/issues/index.md:256]: Then we should rather remove the section as it no longer applies.

File: mountainduck/issues/index.md [hunk 2]
```diff
@@ -253,15 +253,6 @@ It may be that Finder closes the connection because Mountain Duck hasn't answere
...
-<del> The `._*` files contain metadata about the files some applications write in additional to the file content. On macOS, this metadata can be stored alongside the file on the filesystem, but on remote volumes, with no metadata support, an auxiliary file is created to contain this information. You can delete metadata on files using `xattr -d <filename>`. </del>
-
-As of version [2.1](https://mountainduck.io/changelog/), extended attributes are only saved in a temporary location and not stored on the mounted remote volume. If you want to revert to saving extended attributes to the server, enter in a *Terminal.app* window:
-
-    defaults write io.mountainduck fs.filenames.metadata.enable false
-
-If you want to delete metadata files, you can open a *Terminal.app* window and enter `dot_clean -m <folder>`.
```
Comment (dkocher) [inline mountainduck/issues/index.md:263]: Restore previous explanation

```
The `._*` files contain metadata about the files some applications write in additional to the file content. On macOS, this metadata can be stored alongside the file on the filesystem, but on remote volumes, with no metadata support, an auxiliary file is created to contain this information. You can delete metadata on files using `xattr -d <filename>`.
```

**Review by dkocher — CHANGES_REQUESTED** [review]

File: mountainduck/issues/index.md (hunk 2 above)
Comment (Cassie205) [reply mountainduck/issues/index.md:263]: A strikedthrough header doesn't make sens therefore keeping the previous form isn't very userfriendly. As the information about the metadata location is not an actual issue it shoudn't be placed in the issue section

Comment (Cassie205) [reply mountainduck/issues/index.md:263]: > Then we should rather remove the section as it no longer applies.

check previous conversation

**Comment by dkocher** [comment]
Superseded in #481.

**Comment by Leeze555** [comment]
ok

**Pull request closed without merging**